frontend2/src/components/Header.tsx

```diff
@@ -0,0 +1,345 @@
+            <div className="flex flex-shrink-0 items-center">
+              <img className="h-8 w-auto" src={require(`../${logoPath}`)} alt="Battlecode" />
+            </div>
+            <div className="hidden sm:ml-6 sm:block">
+        {/* <div className="flex space-x-4">
+          <a
+            href="#"
+            className={`${
+              activeLink === 'Info' ? 'bg-gray-900 text-white' : 'text-gray-300 hover:bg-gray-700 hover:text-white'
+            } rounded-md px-3 py-2 text-sm font-medium`}
+            onClick={() => {handleLinkClick('info')}}
+            aria-current={activeLink === 'Info' ? 'page' : undefined}
+          >
+            Info
+          </a>
+          <a
+            href="#"
+            className={`${
+              activeLink === 'Team' ? 'bg-gray-900 text-white' : 'text-gray-300 hover:bg-gray-700 hover:text-white'
+            } rounded-md px-3 py-2 text-sm font-medium`}
+            onClick={() => {handleLinkClick('Team')}}
+            aria-current={activeLink === 'Team' ? 'page' : undefined}
+          >
+            Team
+          </a>
+          <a
+            href="#"
+            className={`${
+              activeLink === 'Dev Team' ? 'bg-gray-900 text-white' : 'text-gray-300 hover:bg-gray-700 hover:text-white'
+            } rounded-md px-3 py-2 text-sm font-medium`}
+            onClick={() => {handleLinkClick('Dev Team')}}
+            aria-current={activeLink === 'Dev Team' ? 'page' : undefined}
+          >
+            Dev Team
+          </a>
+          <a
+            href="#"
+            className={`${
+              activeLink === 'Sponsors' ? 'bg-gray-900 text-white' : 'text-gray-300 hover:bg-gray-700 hover:text-white'
+            } rounded-md px-3 py-2 text-sm font-medium`}
+            onClick={() => {handleLinkClick('Sponsors')}}
+            aria-current={activeLink === 'Sponsors' ? 'page' : undefined}
+          >
+            Sponsors
+          </a>
+        </div> */}
+      </div>
+          </div>
+          <div className="absolute inset-y-0 right-0 flex items-center pr-2 sm:static sm:inset-auto sm:ml-6 sm:pr-0">
+            <button
+              type="button"
+              className="relative rounded-full bg-gray-800 p-1 text-gray-400 hover:text-white focus:outline-none focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-gray-800"
+            >
+              <span className="absolute -inset-1.5"></span>
+              <span className="sr-only">View notifications</span>
+              <svg className="h-6 w-6" fill="none" viewBox="0 0 24 24" strokeWidth="1.5" stroke="currentColor" aria-hidden="true">
+                <path strokeLinecap="round" strokeLinejoin="round" d="M14.857 17.082a23.848 23.848 0 005.454-1.31A8.967 8.967 0 0118 9.75v-.7V9A6 6 0 006 9v.75a8.967 8.967 0 01-2.312 6.022c1.733.64 3.56 1.085 5.455 1.31m5.714 0a24.255 24.255 0 01-5.714 0m5.714 0a3 3 0 11-5.714 0" />
+              </svg>
+            </button>
```

Comment:
This notification button doesn't seem to do anything right now, let's remove it.